src/components/modules/api/blocks.ts

```diff
@@ -28,6 +28,7 @@ export default class BlocksAPI extends Module {
       stretchBlock: (index: number, status = true): void => this.stretchBlock(index, status),
       insertNewBlock: (): void => this.insertNewBlock(),
       insert: this.insert,
+      toggleBlockSetting: (openingState?: boolean): void => this.toggleBlockSetting(openingState),
```

Comment:
i think, it is related to `api.toolbar`, not `api.blocks`

Reply:
But its related to current selected block